Integrations/integration-C2sec.yml

```
@@ -0,0 +1,246 @@
+          return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {"C2sec.AddCompany": call.json() }
+                }
+       else:
+            return ('Error adding a new company - status code: [%d] - reason: %s' % (call.status_code, call.text))
+
+
+
+    def queryWorkitem(workitemid):
+       params = {'apikey': API_KEY, 'workitemid': workitemid}
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(ENDPOINTURL + '/iriskqueryapistatus', headers=HEADERS, verify=USE_SSL, params=params)
+       if call.status_code == requests.codes.ok:
+         resp = call.json()
+         resp['apistatus']['workitemid'] = workitemid
+         md = tableToMarkdown('Query WorkItem Result', resp['apistatus'])
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': resp,
+                    'HumanReadable': md,
+                    'EntryContext': {"C2sec.Scan(val.workitemid && val.workitemid == obj.workitemid)": resp['apistatus']}
+                }
+       else:
+           return ('Error querying a workitem [%d] - reason: %s' % (call.status_code, call.text))
+
+    def queryIssues(domain):
+       params = {'apikey': API_KEY, 'domain': domain }
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(ENDPOINTURL + 'iRiskQueryIssues', headers=headers, verify=USE_SSL, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Query Issues Result', call.json()['issueList'])
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {"C2sec.Issues": call.json()['issueList'] }
+                }
+       else:
+           return ('Error querying for issues [%d] - reason: %s' % (call.status_code, call.text))
+
+
+    def rescan(domain):
+       params = {'apikey': API_KEY, 'domain': domain}
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(ENDPOINTURL + 'iRiskRescanCompany', headers=HEADERS, verify=USE_SSL, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Rescan Results', call.json())
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {"C2sec.ScanRequest(val.workitemid && val.workitemid == obj.workitemid)": call.json()}
```

Not sure about the difference between queryWorkitem the context path seems a bit off here
```suggestion
                    'EntryContext': {"C2sec.Scan(val.workitemid && val.workitemid == obj.workitemid)": call.json()}
```

---

fixed